plugin/src/main/groovy/com/github/ksoichiro/eclipse/aar/GenerateTask.groovy

```diff
@@ -392,6 +392,7 @@ target=${extension.androidTarget}
 
         List<String> list = projectDependencies[p]?.collect { it.getQualifiedName().replaceFirst('^:', '') }
         list = list?.findAll { projectNames.find { prj -> prj == it } == null }
+        list = list.unique()
```

Comment:
I think the `list` sometimes becomes `null` and it causes `NullPointerException` on tests.
Could you change this to `list?.unique()`?